ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/momentumBasedController/feedbackController/taskspace/SpatialFeedbackController.java

```diff
@@ -10,20 +10,12 @@
 import us.ihmc.commonWalkingControlModules.controllerCore.command.inverseKinematics.SpatialVelocityCommand;
 import us.ihmc.commonWalkingControlModules.controllerCore.command.virtualModelControl.VirtualModelControlCommand;
 import us.ihmc.commonWalkingControlModules.controllerCore.command.virtualModelControl.VirtualWrenchCommand;
-import us.ihmc.commonWalkingControlModules.controllerCore.data.FBPose3D;
-import us.ihmc.commonWalkingControlModules.controllerCore.data.FBQuaternion3D;
-import us.ihmc.commonWalkingControlModules.controllerCore.data.FBRateLimitedVector6D;
-import us.ihmc.commonWalkingControlModules.controllerCore.data.FBVector3D;
-import us.ihmc.commonWalkingControlModules.controllerCore.data.FBVector6D;
+import us.ihmc.commonWalkingControlModules.controllerCore.data.*;
```

Comment:
You might not have the most up to date version of the formatter, I would hae thought it would not use the wildcard (*) modifier for such a little amount of imports

Reply:
I tried again , but that wildcard has been used again. 
Has the ihmcformatter been changed?
If so, I'll update this later.